### PR TITLE
Removed appc, changed to coming soon

### PR DESCRIPTION
**Pull request by joel-hamill**
## What are your changes?

Removed appc, changed to coming soon
## What type of changes does your doc introduce?
- [x ] Bug fix
- [ ] New feature 
## What is the urgency level of this change?
- [ ] Blocker
- [ x] High
- [ ] Medium
## I have completed these items:
- [ x] I have tested all commands and code snippets.
- [ x] I have built locally and tested for broken links and formatting.
- [ x] I have made this change for all affected versions (e.g. 1.7, 1.8, and 1.9).
- [ x] My doc follows the style guide: https://github.com/dcos/dcos-docs#contributing.

Ping @emanic @sascala @joel-hamill for reviewing pull request changes.

### CODE REVIEW

**Comment by karlkfi** [comment]
Several grammar issues. You may want to read it again.

File: 1.9/usage/containerizers.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ DC/OS supports the Mesos containerizer types [DC/OS Universal container runtime]
 
 # DC/OS Universal container runtime
 
-The [Universal container runtime](http://mesos.apache.org/documentation/latest/container-image) extends the Mesos containerizer to support provisioning [Docker](https://docker.com/) and [AppC](https://github.com/appc/spec) container images. This means that you can use both the Mesos containerizer and other container image types in DC/OS. You can still use the Docker container runtime directly with DC/OS, but the Universal container runtime supports running Docker images without depending on the Docker Engine, which allows for better integration with Mesos.
+The [Universal container runtime](http://mesos.apache.org/documentation/latest/container-image) extends the Mesos containerizer to support provisioning [Docker](https://docker.com/) container images (([AppC](https://github.com/appc/spec) coming soon). This means that you can use both the Mesos containerizer and other container image types in DC/OS. You can still use the Docker container runtime directly with DC/OS, but the Universal container runtime supports running Docker images without depending on the Docker Engine, which allows for better integration with Mesos.
```
Comment (sascala) [inline 1.9/usage/containerizers.md]: looks like there's an extra `(` in the AppC link.

**Review by sascala — APPROVED** [review]
LGTM except for a stray `(`.

**Comment by joel-hamill** [comment]
@sascala do you mean line 15? It's helpful if make your comments inline :)

**Pull request merged**